models/project/board.go

```diff
@@ -255,16 +269,10 @@ func (p *Project) getDefaultBoard(ctx context.Context) (*Board, error) {
 		return &board, nil
 	}
 
-	// represents a board for issues not assigned to one
-	return &Board{
-		ProjectID: p.ID,
-		Title:     "Uncategorized",
-		Default:   true,
-	}, nil
+	return nil, fmt.Errorf("getDefaultBoard: no default board found")
```

Comment:
That's what I worried about robust. If there is no default board (by any mistake), then the users would always see a 500 page, there is no way for them to recover.

Reply:
I understand, but this case should ideally never happen - and if it does, it's a bug which we want for the users to report... 🤔

Reply:
I think it is solved when such cases are checked every midnight by a cron job